### PR TITLE
3066 service view redesign

### PR DESCRIPTION
**Pull request by goreck888**
https://www.figma.com/file/wFzD4qH6SwpfCybte3pLlb/EOSC-DRAFTS?type=design&node-id=2784%3A18603&mode=design&t=x7WgRlLXsZHmuTLl-1

### CODE REVIEW

**Comment by jarekzet** [comment]
Added latest mods to new service view. Added RWD look.

We need to discuss details boxes content.

image

image

**Comment by Karolina9824** [comment]
Review:

General:
[] Add "Explore Compatible Research Products" button
[] Add "Interoperability Guidelines" button
[] Change "Add Favorite" icon
[] Align button "Access the service" to vertical center
[]Change spacing between Header name and body name to 4px
Detail Boxes:
[] Set every link color to #060F7C Tags:
[] Fix badges in tags to hugxhug
Availability and Languages:
[] Change spacing between regions and language from 18px to 8px
[] Change color for Regions, Languages, and chevrons to specified color
[] Move chevrons to "fill the box"
[] Fix dropdowns
Uptime Monitoring:
[] Change color from #7B7979 to #00000
[] Change color from #52A841 to #25792B
[] Change spacing between from 16px to 8px
[] "Show more details" change color to #060F7C
[] "Show more details" align to the left and down with margin 16px
[] Remove description in Status into Uptime monitoring in details section
[] Remove ">" from "Show more details"
[] Add 16px bottom margin
Dependencies:
[] Break "EGI Cloud Container Compute ->" into two lines. We can't have only -> in the second line
[] Add 16px bottom margin
Financial Information:
[] Remove the whole box if there is only "Pricing"
Services Boxes:
[] Add "Show Less" under description
[] Change spacing by clicking "Show Less" to 16px (default)
Other:
[] Change "Open Access" from grey to brand orange open access

**Comment by Karolina9824** [comment]
figma:
https://www.figma.com/file/wFzD4qH6SwpfCybte3pLlb/EOSC-DRAFTS?type=design&node-id=2784%3A18603&mode=design&t=x7WgRlLXsZHmuTLl-1

**Comment by Marcelinna** [comment]
@Karolina9824 @jarekzet 
I dont see EOSC PID: 3d878184 in new view

image

**Comment by Marcelinna** [comment]
@Karolina9824 @jarekzet 
I dont see box with statistics in new view 
image

**Comment by Marcelinna** [comment]
@Karolina9824 @jarekzet 
I don't see in a new view
Categorization, Scientific categorisation, Marketplace Locations: 
image
image

**Comment by Marcelinna** [comment]
@Karolina9824 @agpul @jarekzet we delete favorite functionality?
I do not see this box in Figma, but I see it in applications
image

**Comment by Karolina9824** [comment]
> @Karolina9824 @jarekzet
> I don't see in a new view
> Categorization, Scientific categorisation, Marketplace Locations:
> image
> image

All of these you have on the top in the new design, under header/title in the small blue boxes/badges

**Comment by Karolina9824** [comment]
> @Karolina9824 @jarekzet I dont see EOSC PID: 3d878184 in new view
> 
> image

@jarekzet we could add this in details in free box in the second line

**Comment by Karolina9824** [comment]
> @Karolina9824 @jarekzet
> I dont see box with statistics in new view
> image

we remove them

**Comment by agpul** [comment]
> @Karolina9824 @agpul @jarekzet we delete favorite functionality? I do not see this box in Figma, but I see it in applications image

This functionality was removed from the front, it will be cover in the SS box and UD. We remove the functionality from the details page view as it will be inconsistent in the system (MP landing pages vs OpenAire landing pages)

**Comment by agpul** [comment]
> > @Karolina9824 @jarekzet
> > I dont see box with statistics in new view
> > image
> 
> we remove them

@Karolina9824 why statistics were removed? I think it is quite important feature.

**Comment by Marcelinna** [comment]
@agpul @Karolina9824 @jarekzet 
I dont see [Ask a question about this service?], we want to remove it ?
image

**Comment by Marcelinna** [comment]
@Karolina9824 @agpul @jarekzet 
we want duplication here?
image

**Comment by Karolina9824** [comment]
> @Karolina9824 @jarekzet
> I dont see box with statistics in new view
> image

figma: 
https://www.figma.com/file/wFzD4qH6SwpfCybte3pLlb/EOSC-DRAFTS?type=design&node-id=2784%3A18604&mode=design&t=x7WgRlLXsZHmuTLl-1

**Comment by Karolina9824** [comment]
> @Karolina9824 @agpul @jarekzet
> we want duplication here?
> image

nope

**Comment by Marcelinna** [comment]
@agpul @goreck888 @Karolina9824 
https://marketplace-2.docker-fid.grid.cyf-kr.edu.pl/services/egi-cloud-compute
About tab
Suggested compatible services 
image
Details, Guidelines, Review tab
image


Why on About tab I see much more Suggested compatible services than on Details, Guidelines, Review tab
We want Suggested compatible services on every page? or only on About page?

**Comment by Marcelinna** [comment]
When I click on Contact provider
image
image

**Comment by Marcelinna** [comment]
@Karolina9824 @agpul 
image
I dont see in new view Categorisation

**Comment by Marcelinna** [comment]
@agpul @Karolina9824 

image
For me, it would be better if the language and region were visible like the the rest of the elements, we don't have such expand/collapse fun anywhere, there is a lot of text in Target users too and it looks ok.

**Comment by Karolina9824** [comment]
> > @Karolina9824 @jarekzet I dont see EOSC PID: 3d878184 in new view
> > image
> 
> @jarekzet we could add this in details in free box in the second line
[
](https://www.figma.com/file/wFzD4qH6SwpfCybte3pLlb/EOSC-DRAFTS?type=design&node-id=2784%3A18604&mode=design&t=x7WgRlLXsZHmuTLl-1)

**Comment by Marcelinna** [comment]
@Karolina9824 @agpul we dont show Resource ogranisation and Provider by ?? I think we should
image

**Comment by Karolina9824** [comment]
> When I click on Contact provider
> image
> image

it's suppose to direct to "ask provider a question" modal

**Comment by Karolina9824** [comment]
Changes:
-[] New Statistics Box
-[] New "EOSC PID" Box
-[] Remove "target users" from the "Classification" Box
-[] Standardize "Suggested compatible services" across all tabs
-[] Check the issue with the "Contact provider" popup
-[] Add Organization
-[] Add Provided by
-[] Change the weight of the description from medium to regular
-[] Add "Marketplace Location" in the "classification" box

**Comment by Marcelinna** [comment]
@jarekzet 
image

**Comment by Marcelinna** [comment]
Concat provider is visible only on About tab
image

**Comment by Marcelinna** [comment]
@jarekzet text in the box
image

**Comment by Marcelinna** [comment]
Marketplace location should be bold
image

**Comment by Marcelinna** [comment]
@goreck888 When I add data source, I see less link w right box 
Datasource
image
Service 
image

**Comment by Marcelinna** [comment]
Duplication
image

**Comment by Karolina9824** [comment]
- [ ] show more-show less for "provided by"
- [ ] "provider by" in one line
- [ ] description  text height line150%
- [ ] boxes in the "about" should be according 1.Uptime monitoring 2.Statistics 3. Service Identifiers 4. Availability and Language. Only 4

**Comment by Marcelinna** [comment]
@goreck888 @jarekzet 
Service with interoperability guidelines
image


service without interoperability guidelines
image

**Comment by aleksandrahojszyk** [comment]
Please change description text size to 14px (also @Karolina9824 update figma pls) 
Please change text in statistics:.//////
"Service catalog entry views" -> Catalog views.
Service visits from the service catalog -> Catalog service visits.
Total number of appearances in the project -> Project appearance total.

**Comment by Karolina9824** [comment]
> Please change description text size to 14px (also @Karolina9824 update figma pls)
> Please change text in statistics:.//////
> "Service catalog entry views" -> Catalog views.
> Service visits from the service catalog -> Catalog service visits.
> Total number of appearances in the project -> Project appearance total.

done 
https://www.figma.com/file/wFzD4qH6SwpfCybte3pLlb/EOSC-DRAFTS?type=design&node-id=2784%3A18604&mode=design&t=ZZTRspUNp8sUfY7v-1

**Comment by jarekzet** [comment]
Added latest mods:

1. Description font size:

image

2. Statistics:

image

**Review by Marcelinna — APPROVED** [review]

File: app/javascript/app/controllers/favourite_controller.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default class extends Controller {
     const response = await this.sendRequest(this.updateFavourites());
     const result = await this.getResponse(response);
     this.updateCheckboxLabels();
-    this.showPopup(result);
+    // this.showPopup(result);
```
Comment (wujuu) [inline app/javascript/app/controllers/favourite_controller.js:15]: nit: delete if not needed

File: db/schema.rb [hunk 2]
```diff
@@ -583,7 +583,6 @@
     t.string "links", default: [], array: true
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.string "best_access_right"
```
Comment (wujuu) [inline db/schema.rb:586]: I didn't see a migration for this change

**Review by wujuu — APPROVED** [review]

**Review by Michal-Kolomanski — APPROVED** [review]

File: db/schema.rb (hunk 2 above)
Comment (goreck888) [reply db/schema.rb]: Resolved

File: app/javascript/app/controllers/favourite_controller.js (hunk 1 above)
Comment (goreck888) [reply app/javascript/app/controllers/favourite_controller.js]: Removed

**Comment by Marcelinna** [comment]
https://www.figma.com/file/wFzD4qH6SwpfCybte3pLlb/EOSC-DRAFTS?type=design&node-id=4394-18582&mode=design&t=cpTasmQ4cFH5Jx8O-0

**Pull request merged**